<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ЭПИ Лабораторная работа №6 Соколов</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="pmLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Трудоемкость:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tmLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время разработки: </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="wbsTable">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <row>
    <property name="text">
     <string>Анализ требований</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Проектирование продукта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Программирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Планирование тестирования</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Верификация и аттестация</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Канцелярия проекта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Управление конфигурацией и обеспечение качества</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Создание руководств</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Итого</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Бюджет (%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Человеко-месяцы</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>44</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стандартное распределение работ по видам деятельности WBS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="classicTable">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>540</y>
     <width>561</width>
     <height>231</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Планирование и определение требований</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Проектирование продукта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Детальное проектирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Кодирование и тестирование отдельных модулей</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Интеграция и тестирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Итого без планирования</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Итого</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Работа</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Распределение работ и времени по стадиям жизненного цикла при традиционном подходе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graphsButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>700</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить графики</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string> KLOC (количество строк кода)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="sizeEdit">
      <property name="text">
       <string>55</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Режим проекта</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_16">
      <item>
       <property name="text">
        <string>Обычный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Промежуточный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Встроенный</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Атрибуты программного продукта</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DATA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>(Требуемая надежность)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>(Размер базы данных)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RELY</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_1">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_2">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>CPLX</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>(Сложноть продукта)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_3">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Атрибуты компьютера</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>VIRT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_6">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TIME</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pmLabel_3">
       <property name="text">
        <string>(Изменчивость виртуальной машины)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_5">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_4">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>STOR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>(Ограничение времени выполнения)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="pmLabel_2">
       <property name="text">
        <string>(Ограничение объема основной памяти)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TURN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox_7">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="pmLabel_4">
       <property name="text">
        <string>(Время реакции компьютера)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>501</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Атрибуты персонала</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_8">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>VEXP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ACAP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="pmLabel_8">
       <property name="text">
        <string>(Знание виртуальной машины)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>PCAP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>AEXP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_9">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_10">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox_11">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pmLabel_7">
       <property name="text">
        <string>(Способности программиста)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="pmLabel_6">
       <property name="text">
        <string>(Знание приложений)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="pmLabel_5">
       <property name="text">
        <string>(Способности аналитика) </string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>LEXP</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="pmLabel_9">
       <property name="text">
        <string>(Знание языка программирования)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="comboBox_12">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Атрибуты проекта</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QLabel" name="pmLabel_10">
       <property name="text">
        <string>(Использование современных методов)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_13">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>MODP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>TOOL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_14">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="pmLabel_11">
       <property name="text">
        <string>(Использование программных инструментов)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>SCED</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_15">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Очень низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Низкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номинальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высокий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очень высокий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pmLabel_12">
       <property name="text">
        <string>(Требуемые сроки разработки)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать проект</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Средняя зарплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="costLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>140</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Приблизительный бюджет:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="costEdit">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
